### PR TITLE
LPS 87199 As a user, I want to Find answers through the FAQ

### PR DESCRIPTION
**Pull request by camisLuisa**

### CODE REVIEW

File: android/forms-screenlet-demo/app/src/main/res/layout/activity_faq_list.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		android:id="@+id/faq_list_toolbar"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:background="@color/white"
+		android:elevation="@dimen/toolbar_elevation"
+		app:title="@string/faq"
+		app:titleTextColor="@color/toolbar_text_color"
+		/>
+
+		<TextView
```
Comment (phcp) [inline android/forms-screenlet-demo/app/src/main/res/layout/activity_faq_list.xml]: SF

File: android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+				android:textSize="@dimen/help_text_size"
+				android:layout_marginBottom="@dimen/help_text_margin_bottom"
+
+				/>
+
+			<View
+				android:layout_width="match_parent"
+				android:layout_height="@dimen/help_line_height"
+				android:layout_gravity="bottom"
+				android:background="@color/help_line_color"/>
```
Comment (phcp) [inline android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml]: SF

File: android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+			>
+
+			<TextView
+				android:layout_width="wrap_content"
+				android:layout_height="wrap_content"
+				android:text="@string/contact_support"
+				android:textStyle="normal"
+				android:textSize="@dimen/help_text_size"
+				android:layout_marginBottom="@dimen/help_text_margin_bottom"
+
```
Comment (phcp) [inline android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml]: Remove extra space.

File: android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+				/>
+
+			<View
+				android:layout_width="match_parent"
+				android:layout_height="@dimen/help_line_height"
+				android:layout_gravity="bottom"
+				android:background="@color/help_line_color"/>
```
Comment (phcp) [inline android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml]: SF

File: android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqActivity.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+import com.liferay.mobile.screens.thingscreenlet.screens.views.Detail;
+import com.liferay.mobile.screens.util.AndroidUtil;
+import kotlin.Unit;
+
+/**
+ * @author Luísa Lima
+ */
+public class FaqActivity extends AppCompatActivity {
+
+	public static final String THING_ID_EXTRA = "thingId";
```
[the remark this answers is not in the document]
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqActivity.java]: Okay

File: android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqActivity.java [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+	protected void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		setContentView(R.layout.activity_faq);
+
+		faqContent = findViewById(R.id.faq_content);
+		toolbar = findViewById(R.id.faq_toolbar);
+		setSupportActionBar(toolbar);
+		getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+		getSupportActionBar().setDisplayShowHomeEnabled(true);
+
```
[the remark this answers is not in the document]
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqActivity.java]: okay

File: android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqActivity.java [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+	}
+
+	@Override
+	public boolean onSupportNavigateUp() {
+		onBackPressed();
+		return true;
+	}
+
+	private void loadResource() {
+		String url = getIntent().getStringExtra(THING_ID_EXTRA);
```
[the remark this answers is not in the document]
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqActivity.java]: Okay

File: android/forms-screenlet-demo/app/src/main/AndroidManifest.xml [hunk 8]
```diff
@@ -57,6 +57,12 @@
 		</activity>
 		<activity android:name="com.liferay.mobile.formsscreenletdemo.view.WorkflowActivity">
 		</activity>
+		<activity android:name="com.liferay.mobile.formsscreenletdemo.view.HelpActivity">
+		</activity>
+		<activity android:name="com.liferay.mobile.formsscreenletdemo.view.FaqListActivity">
```
[the remark this answers is not in the document]
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/AndroidManifest.xml]: Okay

File: android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqActivity.java [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+import com.liferay.mobile.screens.rating.RatingScreenlet;
+import com.liferay.mobile.screens.thingscreenlet.screens.ThingScreenlet;
+import com.liferay.mobile.screens.thingscreenlet.screens.views.Detail;
+import com.liferay.mobile.screens.util.AndroidUtil;
+import kotlin.Unit;
+
+/**
+ * @author Luísa Lima
+ */
+public class FaqActivity extends AppCompatActivity {
```
[the remark this answers is not in the document]
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqActivity.java]: okay

File: android/forms-screenlet-demo/app/src/main/res/layout/activity_faq_list.xml (hunk 1 above)
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/res/layout/activity_faq_list.xml]: okay

File: android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml (hunk 2 above)
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml]: okay

File: android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml (hunk 3 above)
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml]: okay

File: android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml (hunk 4 above)
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/res/layout/activity_help.xml]: okay

File: android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqActivity.java [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+
+	public static final String THING_ID_EXTRA = "thingId";
+	private ThingScreenlet faqContent;
```
[the remark this answers is not in the document]
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqActivity.java]: Okay, I'll do it.

File: android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqListActivity.java [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+import kotlin.Unit;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * @author Luísa Lima
+ */
+public class FaqListActivity extends AppCompatActivity implements ScreenletEvents {
+
+	private ThingScreenlet faqPopularList;
```
[the remark this answers is not in the document]
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqListActivity.java]: okay

File: android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqListActivity.java [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+import com.liferay.mobile.screens.thingscreenlet.screens.views.Row;
+import com.liferay.mobile.screens.thingscreenlet.screens.views.Scenario;
+import kotlin.Unit;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * @author Luísa Lima
+ */
+public class FaqListActivity extends AppCompatActivity implements ScreenletEvents {
```
[the remark this answers is not in the document]
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FaqListActivity.java]: okay

**Pull request merged**